R/tt_pos_and_access.R

```diff
@@ -1352,17 +1352,3 @@ setMethod("tail", "VTableTree",
                      reindex_refs = reindex_refs)
     res
 })
-
-
-#' @rdname brackets
-#' @name bracket_subsetting
-#' @export
-#' @usage
-#' x[i, j, ...]
-base::`[`
-
-#' @rdname brackets
-#' @export
-#' @usage
-#' x[i, j, ...] <- value
```

Comment:
Have you visually checked the pkgdown since you made this change, @Melkiades? I vaguely remember that may have been why we did this. @shajoezhu

Reply:
You are very right! Now, I remember that, and it is true, the methods are automatically printed with the whole signature, being less than readable. I think though to have found a solution. See comment below